spring-boot/admin/src/main/resources/templates/admin/fragments/api-keys/consumer/list.html

```diff
@@ -83,6 +62,7 @@ <h2 th:text="#{admin.api-keys-consumer.subtitle}">[Post Administration</h2>
                                         </th>
                                         <th th:if="${testService == null}" th:text="#{admin.api-keys.list.col.created.heading}">Created </th>
                                         <th th:if="${testService == null}" th:text="#{admin.api-keys.list.col.validity.heading}">Validity</th>
+                                        <th></th>
```

Comment:
I would suggest to add some column heading here, either "Delete" or a more generic word like "Action"?

Another alternative would be to keep the empty heading, but then remove the "sort-by button" in the column header

Reply:
I preferred the second option. The heading is empty and the sort-by-button was removed. Done.